OracleFMWInfrastructure/dockerfiles/12.2.1.3/README.md

```diff
@@ -101,31 +101,30 @@ In this image the domain home will be persisted to a volume in the host. The "-v
 
 You can override the default values of the following parameters during runtime in the `./properties/domain.properties` file. The script `./container-scripts/setEnv.sh` sets the environment variables to configure the domain. The default values of the environment variables are:
 
-      * `ADMIN_NAME`                  (default: `AdminServer`)
-      * `ADMIN_LISTEN_PORT`           (default: `7001`)
-      * `ADMIN_HOST`                  (default: `InfraAdminContainer`)
-      * `DOMAIN_NAME`                 (default: `infra_domain`)
-      * `ADMINISTRATION_PORT_ENABLED` (default: `true`)
-      * `ADMINISTRATION_PORT`         (default: `9002`)
-      * `MANAGED_NAME`                (default: `infraServer1`)
-      * `MANAGEDSERVER_PORT`          (default: `8001`)
-      * `RCUPREFIX`                   (default: `INFRA01`)
-      * `PRODUCTION_MODE`             (default: `prod`)
+	`DOMAIN_NAME=myinfraDomain`
```

Comment:
Instead of using single backticks for each line, I suggest using a full code block, i.e. using triple backticks to open and close the block.